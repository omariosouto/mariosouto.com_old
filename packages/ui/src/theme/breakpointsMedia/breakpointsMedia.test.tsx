import React from 'react';
import styled from 'styled-components';
import { screen, render } from '../../infra/tests/testUtils';

test('to be or not be', () => {
  expect(true).toBe(true);
});

test('component', () => {
  const Title = styled.h1`
    color: ${({ theme }) => theme.colors.primary.main.color};
  `;
  const Component = () => (
    <div data-testid="test">
      <Title>Test</Title>
    </div>
  );
  render(<Component />);
  expect(screen.getAllByTestId('test')).toMatchSnapshot();
});
