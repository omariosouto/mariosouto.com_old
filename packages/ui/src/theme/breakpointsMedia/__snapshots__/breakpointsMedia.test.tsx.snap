// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component 1`] = `
Array [
  .c0 {
  color: var(--colors_primary_main_color);
}

<div
    data-testid="test"
  >
    <h1
      class="c0"
    >
      Test
    </h1>
  </div>,
]
`;
